pytket/pytket/qasm/qasm.py

```diff
@@ -370,7 +370,7 @@ def barr(self, tree: List[Arg]) -> Iterable[CommandDict]:
                 signature.append("Q")
             else:
                 raise QASMParseError(
-                    "UnitID", arg, "in Barrier arguments is not declared."
+                    "UnitID " + arg + " in Barrier arguments is not declared."
```

Comment:
Doesn't `arg` need to be converted to a string here?

Reply:
hmm yeah, done